packages/curve-ui-kit/src/features/user-profile/UserProfile.tsx

```diff
@@ -0,0 +1,25 @@
+import Box from '@mui/material/Box'
+import Drawer from '@mui/material/Drawer'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+
+import { Home } from './home'
+
+type Props = {
+  open: boolean
+  onClose: () => void
+}
+
+export const UserProfile = ({ open, onClose }: Props) => (
+  <Drawer open={open} anchor="right" onClose={onClose}>
+    <Box
+      sx={{
```

Comment:
The Drawer already renders a `MuiPaper` (but by default that uses Layer[2])
```suggestion
  <Drawer open={open} anchor="right" onClose={onClose} PaperProps={{ sx: {
```